.github/workflows/execution-plan-main.yml

```diff
@@ -13,6 +13,12 @@ jobs:
       trigger: main
     secrets: inherit
 
+  integration-tests:
```

Comment:
for testing purposes, add this to execution-plan-pr.yaml
I

Reply:
I did this in the previous commit, I tested with an already pushed tag.
I moved it from the PR workflow for these reasons:
- Having an integration test every time you push to a PR sounds really nice but it takes quite a while to run this workflow. I think this should be something discussed as a team.
- But more importantly: the image is not built & pushed in the PR workflow, only when merging with main. So it's not possible to use `make version` yet... and building the image every time you push to a PR would make the 1st point even worse.

Reply:
@jaypontes ok fair.
Could you link a successful run here?

I would still keep it in *for now*, and remove it again before you merge it. You need some way to show that it works before merging.

Since this PR is still a draft, I assume you want to make further changes still.

Reply:
Here it is: https://github.com/freiheit-com/kuberpult/actions/runs/4136581181/jobs/7150679117 😁